android/src/main/java/com/doublesymmetry/trackplayer/extensions/AudioPlayerStateExt.kt

```diff
@@ -13,5 +13,7 @@ val AudioPlayerState.asLibState: State
             AudioPlayerState.PLAYING -> State.Playing
             AudioPlayerState.IDLE -> State.Idle
             AudioPlayerState.ENDED -> State.Stopped
+            AudioPlayerState.ERROR -> State.Stopped
```

Comment:
Yes I reviewed this pr. Currently some behavior is changing in a new version. For example, the output of the getState method changes. **It now returns { state, error }** instead of returning you a **string**. I think this pr is important so that historical users can continue to use it **without the need for code changes.**

Reply:
@HyopeR we just merged the PR I mentioned above. If you want to fix this in the `3.2.x` version of the library please rebase your PR against this new branch: https://github.com/doublesymmetry/react-native-track-player/tree/v3.2

It would also be great to see a PR for `main` as that is what we're going to be using moving forward.